javatests/google/registry/rde/imports/RdeHostLinkActionTest.java

```diff
@@ -87,9 +89,17 @@ public void test_mapreduceSuccessfullyLinksHost() throws Exception {
     Key<DomainResource> superOrdinateDomainKey = Key.create(superordinateDomain);
     pushToGcs(DEPOSIT_1_HOST);
     runMapreduce();
+    // verify that host is linked to domain
     List<HostResource> hosts = ofy().load().type(HostResource.class).list();
     assertThat(hosts).hasSize(1);
     assertThat(hosts.get(0).getSuperordinateDomain()).isEqualTo(superOrdinateDomainKey);
+    // verify that last superordinate change is within the last 10 seconds
+    assertThat(hosts.get(0).getLastSuperordinateChange())
+        .isAtLeast(DateTime.now().minus(Seconds.seconds(10)));
```

Comment:
Looks like I need to inject the time more than once, with the second time being slightly in the future to avoid triggering the error here:
```
/** Check that the timestamp of each BackupGroupRoot is in the past. */
  private void checkBackupGroupRootTimestamps(
      DateTime transactionTime, Set<Entry<Key<BackupGroupRoot>, BackupGroupRoot>> bgrEntries) {
    ImmutableMap.Builder<Key<BackupGroupRoot>, DateTime> builder = new ImmutableMap.Builder<>();
    for (Entry<Key<BackupGroupRoot>, BackupGroupRoot> entry : bgrEntries) {
      DateTime updateTime = entry.getValue().getUpdateAutoTimestamp().getTimestamp();
      if (!updateTime.isBefore(transactionTime)) {
        builder.put(entry.getKey(), updateTime);
      }
    }
    ImmutableMap<Key<BackupGroupRoot>, DateTime> problematicRoots = builder.build();
    if (!problematicRoots.isEmpty()) {
      throw new TimestampInversionException(transactionTime, problematicRoots);
    }
  }
```